logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/CreateGroupConversationUseCase.kt

```diff
@@ -20,11 +25,15 @@ import com.wire.kalium.util.DateTimeUtil
  * Will wait for sync to finish or fail if it is pending,
  * and return one [Result].
  */
+@Suppress("LongParameterList")
 class CreateGroupConversationUseCase internal constructor(
     private val conversationRepository: ConversationRepository,
     private val conversationGroupRepository: ConversationGroupRepository,
     private val syncManager: SyncManager,
-    private val currentClientIdProvider: CurrentClientIdProvider
+    private val currentClientIdProvider: CurrentClientIdProvider,
+    private val selfUserId: UserId,
+    val persistMessage: PersistMessageUseCase,
```

Comment:
```suggestion
    private val persistMessage: PersistMessageUseCase,
```

If necessary, please make `PersistMessageUseCase` internal as well.

It shouldn't be public in the first place, and from what I've seen, making it `internal` won't cause any breaking changes